src/maps/filters.ts

```diff
@@ -57,9 +57,9 @@ function toIsNull(value?: boolean): IsNull | undefined {
   return { is_null_: value }
 }
 
-function toLike(value?: string): Like | undefined {
-  if (typeof value === 'undefined') {
-    return value
+function toLike(value?: string | null): Like | undefined {
+  if (!value) {
+    return undefined
```

Comment:
I noticed that when binding a `_like` value to one of our inputs we can end up with both `null` and `''` as empty values. This change makes sure no falsey values are sent to the api which makes for more accurate subscription caching.